src/hotspot/share/classfile/verifier.cpp

```diff
@@ -134,7 +134,9 @@ void Verifier::trace_class_resolution(Klass* resolve_class, InstanceKlass* verif
 }
 
 // Prints the end-verification message to the appropriate output.
-void Verifier::log_end_verification(outputStream* st, const char* klassName, Symbol* exception_name, TRAPS) {
+void Verifier::log_end_verification(outputStream* st, const char* klassName, Symbol* exception_name) {
+   Thread* THREAD = Thread::current();
+
```

Comment:
This one would be better if you passed THREAD->pending_exception() as an oop.  Then you wouldn't have to have HAS_PENDING_EXCEPTION.

Reply:
This is logging so I guess it doesn't really matter.

Reply:
Changed to passing it as an oop.

Reply:
that looks good!